tests/thingpedia-integration/run-integration.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -ex
+srcdir=`dirname $0`/../..
+srcdir=`realpath $srcdir`
+cd $srcdir 
+
+# Run thingpedia-integration test inside frontend pod
+POD=`kubectl get pod -l app=frontend -o jsonpath="{.items[0].metadata.name}"`
+kubectl exec $POD --  bash -c "cd /opt/almond-cloud && npx nyc tests/thingpedia-integration/thingpedia-integration.sh"
+
+# Run selenium test on ubuntu with Firefox installed
+THINGENGINE_CONFIGDIR=tests/thingpedia-integration/k8s npx nyc ts-node tests/test_website_selenium.js
+
+# Copy coverage outputs from frontend pod
+kubectl cp $POD:/opt/almond-cloud/.nyc_output .nyc_output
```

Comment:
Is /opt/almond-cloud writable from inside the test docker? Do we get any coverage at all from this process?

And what about Go coverage?

Reply:
It is in the frontend pod since I enabled root.  I think these coverage outputs are from the thingpedia  integration tests.
Go coverage is only enabled in the GO unittests.

Reply:
Yeah coverage of the test code itself is not very interesting. It would be nice to enable coverage of the actual frontend process. Not sure how we do that, and how we kill the frontend and capture it...

Reply:
Got the coverage outputs of frontend and worker.  Now the coverage report looks reasonable.